TLC5955.h

```diff
@@ -73,14 +73,18 @@ void setChannel(uint16_t channel_number, uint16_t value);
 
 /* Get LED Intensities */
 uint16_t getChannelValue(uint16_t channelNum, int color_channel_index);
+void getLedCurrents(float* currents, uint16_t* gs);
```

Comment:
I generally think that you should use double unless you are trying to hit performance. 

Also, do you want to do everything in A, not mA? It makes things easier.